integration_tests/mockApis/manageUsers.ts

```diff
@@ -21,23 +21,17 @@ export default {
       },
     }),
 
-  stubUserDetails: (args: { name: User['name']; username: User['username'] }): SuperAgentRequest =>
+  stubUserDetails: (args: { user: User }): SuperAgentRequest =>
```

Comment:
I think this can be simplified to `stubUserDetails: (user: User ):` if there's only one thing, like other tests